lab.R

```diff
@@ -18,12 +18,16 @@
 
 ## Setup
 ## **IMPORTANT**: Add all dependencies to `DESCRIPTION`
+install.packages("dplyer")
 library(tidyverse)
+library(tidyselect)
 library(skimr)
 library(visdat)
-
+library(dplyr)
```

Comment:
tidyselect and dplyr should be loaded when you call `library(tidyverse)`.  Because the problems disappeared when you restarted R, I'm guessing you weren't working on this in a clean session, and had packages loaded from other work.  In my experience, the MASS package is used by a lot specialized modeling packages and has a lot of conflicts with dplyr.